5-logging/bookshelf/__init__.py

```diff
@@ -70,15 +71,17 @@ def logout():
     def index():
         return redirect(url_for('crud.list'))
 
-    # Add an error handler. This is useful for debugging the live application,
-    # however, you should disable the output of the exception for production
-    # applications.
+    # Add an error handler that reports exception to Stackdriver Error
+    # Reporting. Note that this Error Handler is only used when Debug
+    # is False
     @app.errorhandler(500)
     def server_error(e):
+        client = error_reporting.Client(app.config['PROJECT_ID'])
```

Comment:
We should probably remove the use of `project_id` from this project at some point.